readthedocs/doc_builder/backends/sphinx.py

```diff
@@ -232,9 +232,7 @@ def build(self):
         self.clean()
         project = self.project
         build_command = [
-            'python',
-            self.python_env.venv_bin(filename='sphinx-build'),
-            '-T',
+            'python', '-m', 'sphinx', '-T',
```

Comment:
This is missing around `python`

https://github.com/readthedocs/readthedocs.org/blob/fb01c6d9d78272e3f4fd146697e8760c04e4fbb6/readthedocs/doc_builder/backends/mkdocs.py#L271-L271

Reply:
Good catch, I wonder if there are other places with the same problem.

Is it the case that this file is always run with the same python as the venv one? If so, can we use `sys.executable` here?

Reply:
@eric-wieser we don't call to `activate`, that's why we need to full binary path here.

Reply:
Fair enough. Updated.